packaging/rpm/keysinuse.spec

```diff
@@ -6,7 +6,7 @@ Release:     LABEL
 Summary:     KeysInUse solution for OpenSSL
 
 Group:       Productivity/Security
-License:     Apache
+License:     MIT
 URL:         www.microsoft.com
```

Comment:
I think this URL should point back to this git repo: `https://github.com/microsoft/KeysInUse-OpenSSL`